Automattic/jetpack
# AI Assistant: Keep content when try again in a transformed block

Fixes #

## Proposed changes:

* Add `originalContent` and `originalMessages` attributes.
* Reset to `originalContent` and `originalMessages`, if they were not defined, it will be undefined.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add a new paragraph
* Use the new AI Extension and transform it to a AI Assistant block through the `Ask AI Assistant` button.
* Iterate and click in Try Again.
* It should reset the content to original paragraph.
